content/_help/verify-your-identity/accepted-state-issued-identification._en.md

```diff
@@ -1,33 +1,31 @@
 ---
 layout: help
-title: Accepted state-issued identification
+title: Accepted identification documents
 category: verify-your-identity
 permalink: /help/verify-your-identity/accepted-state-issued-identification/
 order: 2
 redirect_from:
   - /en/help/verify-your-identity/accepted-state-issued-identification/
+do_list: 
+  -  "**Driver’s license** from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, American Samoa, Mariana Islands and Puerto Rico)."
+  - "**Non-driver’s license state-issued ID card.** This is an identity document issued by the state, the District of Columbia (DC), or US territory that asserts identity but does not give driving privileges."
+dont_list:
+  - You cannot pass identity verification if your ID is expired.
+  - You cannot use extension documents in place of an unexpired&#160;ID.
+  - You cannot use a paper or temporary ID.
+alert: <strong>If you do not have a valid drivers' license or state ID card, you cannot use Login.gov for identity verification.</strong> Please contact the partner agency’s help center to find out what you can do instead."
 ---
 
-At this time, only the following valid state-issued identification is accepted: 
+{% include components/icon-list.html items=page.do_list size='md' icon_color='green' icon_shape='check_circle'%}
 
-* Driver’s license from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, American Samoa, Mariana Islands and Puerto Rico)
-* A non-driver’s license state-issued ID card
+<p class="font-heading-md text-bold">
```

Comment:
@aduth and @kellular , the card said that we need a h3 element here, I cannot skip to h3 without a h2 element, otherwise it fails the accessibility test.

@aduth what's your suggestion on this?

Reply:
@dawei-nava Could you call the text an h2 element but style the text as an h3?

Reply:
Yeah, my first preference would be to use the correct heading order and let the natural styles for that heading take effect.

If we want to stylize the heading to be smaller, I'd go with what @kellular suggested, using the correct heading level, and then applying a `font-heading-*` utility class:

https://designsystem.digital.gov/utilities/font-size-and-family/#utility-font

Reply:
Just to close the loop - Dawei pinged me on Slack about this; we're good to just use an h2, rather than style this as an h3

Reply:
@aduth and @kellular , updated with h2.